### ISSUE TITLE
[REVIEW] SaptarshiSarkar12

### ISSUE THREAD

**Issue by SaptarshiSarkar12** [EddieHubCommunity/support#6742]
### GitHub Profile Link

https://github.com/SaptarshiSarkar12/

**Comment by priyankarpal** [comment]
looks great. 

Suggestions...
- Pin at least 2 or 4 repo

**Comment by SaptarshiSarkar12** [comment]
Thank you for reviewing my GitHub profile @priyankarpal ! I have updated my profile according to your suggestion. 
Will it be reviewed by @eddiejaoude to provide more suggestions?

**Comment by priyankarpal** [comment]
> Thank you for reviewing my GitHub profile @priyankarpal! I have updated my profile according to your suggestion. Will it be reviewed by @eddiejaoude to provide more suggestions?

yeah! you will get a notification when your profile gets reviewed.

**Comment by SaptarshiSarkar12** [comment]
> > Thank you for reviewing my GitHub profile @priyankarpal! I have updated my profile according to your suggestion. Will it be reviewed by @eddiejaoude to provide more suggestions?
> 
> yeah! you will get a notification when your profile gets reviewed.

Okay thank you @priyankarpal !
